pxtblocks/plugins/flyout/flyoutButton.ts

```diff
@@ -62,10 +68,11 @@ export class FlyoutButton extends Blockly.FlyoutButton {
         const iconColor = def["web-icon-color"];
 
         if (icon || iconClass) {
+            const extraIconClass = Object.keys(brandIcons).includes(icon) ? 'brandIcon' : ''
             const svgIcon = Blockly.utils.dom.createSvgElement(
                 'text',
                 {
-                    'class': iconClass ? 'blocklyFlyoutLabelIcon ' + iconClass : 'blocklyFlyoutLabelIcon',
+                    'class': `${iconClass ? 'blocklyFlyoutLabelIcon ' + iconClass : 'blocklyFlyoutLabelIcon'} ${extraIconClass}`,
```

Comment:
Maybe this might be easier to read?
```suggestion
                    'class': `${iconClass ? iconClass : ''} blocklyFlyoutLabelIcon ${extraIconClass}`,
```

Reply:
Tweaked.